saleor/giftcard/notifications.py

```diff
@@ -11,8 +18,16 @@ def send_gift_card_notification(user, app, email, gift_card, manager):
             "app_id": app.id if app else None,
             "app_name": app.name if app else None,
         },
+        "recipient": {
+            "name": _get_recipient_name(email),
+        },
         "recipient_email": email,
```

Comment:
Since we add "recipient" key to payload, should we move the related "recipient_email" to have them in one dict?
```
"recipient": {
    "name": _get_recipient_name(email),
    "email": email
},
```

Or is the `recipient_email` key expected exactly in this form somewhere down the road?

Reply:
👍